examples/booking_app/pubspec.lock

```diff
@@ -660,37 +717,41 @@ packages:
     dependency: "direct main"
     description:
       name: rflutter_alert
-      sha256: a77c8c937fad29e70565388c5934f678b92325047663367b985570ba101929fa
+      sha256: "8ff35e3f9712ba24c746499cfa95bf320385edf38901a1a4eab0fe555867f66c"
       url: "https://pub.dev"
     source: hosted
-    version: "2.0.4"
+    version: "2.0.7"
   rx_bloc:
     dependency: "direct main"
     description:
-      path: "../../packages/rx_bloc"
-      relative: true
-    source: path
+      name: rx_bloc
+      sha256: "22919d1c30b64702c63ff67f0c9a6ca615e43e637022544c6d14f0297fd37afa"
+      url: "https://pub.dev"
+    source: hosted
     version: "4.0.1"
   rx_bloc_generator:
     dependency: "direct dev"
     description:
-      path: "../../packages/rx_bloc_generator"
-      relative: true
-    source: path
+      name: rx_bloc_generator
+      sha256: e51d83485ace36e6bf5bb572069a251c316abef0ab65329743dc5a88f4936730
+      url: "https://pub.dev"
+    source: hosted
     version: "6.0.2"
```

Comment:
The code in the `rxb.g.dart` files seems to be generated with the latest version of the `rx_bloc` dependencies, but the versions here are not the latest. I think this happens because the dependency overrides are removed and `^3.2.0` is specified for `rx_bloc_test` in the `pubspec.yaml`